changes/1814.misc.rst

```diff
@@ -0,0 +1 @@
+Added support for absolute CSS font size keywords in Android, Cocoa, GTK, iOS, and Windows.
```

Comment:
`misc` change notes are hidden in the release notes. This is a publicly-visible feature, so the file should be renamed to `1814.feature.rst`.